posthog/api/feature_flag.py

```diff
@@ -169,6 +172,10 @@ def _update_filters(self, validated_data):
         if "get_filters" in validated_data:
             validated_data["filters"] = validated_data.pop("get_filters")
 
+        active = validated_data.get("active", None)
+        if active is True:
```

Comment:
What language have you been writing in recently? 😋 

Minor nit:

```suggestion
        if active:
```

Reply:
LOL I had this written another way at first like `is not None` and when I did the inverse I explicitly wrote it out...